week2/practice-exercises/3-party-time/script.js

```
@@ -1,16 +1,26 @@
-
 /**
  * 3: Party time
- * 
+ *
  * After reading the documentation make a request to https://reservation100-sandbox.mxapps.io/rest-doc/api
  * and print the response to the console. Use async-await and try/catch.
- * 
+ *
  * Hints:
  * - make sure to use the correct headers and http method in the request
  */
-
-function makeReservation() {
+import fetch from 'node-fetch';
+async function makeReservation() {
   // YOUR CODE GOES IN HERE
+  const apiUrl = 'https://reservation100-sandbox.mxapps.io/api/reservations';
```

This should be wrapped in a try/catch.
Otherwise if something goes wrong the script will fail silently. 😄